js/tji-datasets-explore.js

```diff
@@ -390,17 +447,64 @@ TJIChartView.prototype.filter_data = function() {
     return true;
   })
 
-  this.update_charts(data);
+  this.state.filtered_data = data;
 }
 
-TJIChartView.prototype.update_charts = function(data) {
-  this.update_record_count(data);
+TJIChartView.prototype.update_charts = function() {
+  var that = this;
```

Comment:
I liked update_record_count being coupled to update_charts in that if charts are updated, the record counts should also be updated ... same with create. Can we keep it as it was, where update and create would call the appropriate record count functions.